sys/linux/io_uring.txt

```diff
@@ -96,20 +96,19 @@ io_uring_files_update {
 	fds	ptr64[in, array[fd]]
 }
 
-# NEED: part of fields are input here and part are output. We can't express this yet (#245).
 io_uring_params {
-# sq_entries, cq_entries, features, wq_fd, sq_off, and cq_off are set by the kernel
-	sq_entries	const[0, int32]
-	cq_entries	int32[0:IORING_MAX_CQ_ENTRIES]
-	flags		flags[io_uring_setup_flags, int32]
-	sq_thread_cpu	int32[0:3]
-	sq_thread_idle	int32[0:1000]
-	features	const[0, int32]
-	wq_fd		fd_io_uring[opt]
+	sq_entries	const[0, int32]	(out)
+	cq_entries	int32[0:IORING_MAX_CQ_ENTRIES]	(inout)
+	flags		flags[io_uring_setup_flags, int32]	(in)
+	sq_thread_cpu	int32[0:3]	(in)
+	sq_thread_idle	int32[0:1000]	(in)
+	features	const[0, int32]	(out)
```

Comment:
Does this need to be 0? If not, use `int32	(out)`

Reply:
[fixed](https://github.com/google/syzkaller/pull/2008/files#diff-2e824a90609ef73cc6319dbc406b6b5eR100-R111)